src/amo/reducers/home.js

```diff
@@ -220,6 +237,21 @@ const createInternalAddons = (
   return response.results.map((addon) => createInternalAddon(addon, lang));
 };
 
+export const createInternalPrimaryHeroShelfExternalAddon = (
+  external: PrimaryHeroShelfExternalAddonType,
```

Comment:
The naming confused me here at first v.s the `external` property inside the addon but I get that `Internal` and `External` are the terms used consistently with reducers in addons-frontend.  It's just unfortunate.

Reply:
Yeah, when working on this I recognized that some of these names are quite a mouthful, but I think they accurately reflect what these things are.